week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,29 @@
+let btnBlue = document.querySelector("#blueBtn");
+btnBlue.addEventListener("click", ChangeColor);
+function ChangeColor() {
+  document.querySelector(".jumbotron").style.background = "#588FBD";
+  document.querySelector(".buttons .btn-primary").style.background = "#FFA500";
+  document.querySelector(".buttons .btn-secondary").style.background = "black";
+  document.querySelector(".buttons .btn-secondary").style.color = "white";
+}
+let orangeBtn = document.querySelector("#orangeBtn");
+orangeBtn.addEventListener("click", ChangeColor1);
+function ChangeColor1() {
+  document.querySelector(".jumbotron").style.background = "#f0ad4e";
+  document.querySelector(".buttons .btn-primary").style.background = "#5751fd";
+  document.querySelector(".buttons .btn-secondary").style.background =
+    "#31b0d5";
+  document.querySelector(".buttons .btn-secondary").style.color = "white";
+}
+let grnBtn = document.querySelector("#greenBtn");
+grnBtn.addEventListener("click", ChangeColor2);
+function ChangeColor2() {
+  document.querySelector(".jumbotron").style.background = "#87ca8a";
+  document.querySelector(".buttons .btn-primary").style.background = "black";
+  document.querySelector(".buttons .btn-secondary").style.background =
+"#8c9c08";
+}
+Collapse
+
```

Comment:
Great job Tayfun 👏! I think you have only missed Part 2 of 3-project:

> When the submit button is pressed, it should check that all the form fields are valid

Reply:
Hi Antagoni, we have done it together in a call with two other colleagues and I forgot to update mine. Interestingly, once I update my repository, I am losing previous weeks' works. To overcome that, I am switching to the relevant bransh on github desktop. Unexpectedly, I have week5 branch here but in github desktop!! Not sure how can I amend the files wrt your comments